raiden/api/validations/light_client_safe_operation.py

```diff
@@ -0,0 +1,34 @@
+from http import HTTPStatus
+
+import structlog
+import traceback
+
+from raiden.api.validations.api_error_builder import ApiErrorBuilder
+from raiden.api.validations.light_client_authorization import requires_api_key
+from raiden.exceptions import InsufficientFunds
+
+log = structlog.get_logger(__name__)
+
+
+def lc_safe_operation(func):
```

Comment:
is_safe_operation sounds like "this function will ceck for something and executes if that condition met" 

but in fact, this is a wrapper decorator that executes a function always and catch the exception

i think that `execute_lc_onchain_operation` is better

not a blocker anyway

Reply:
Actually we should call it something like `lc_request` or `handled_lc_request` because actually we don't want to pull down the hub on any lc operation, and you can use this decorator for other operations in the future that are not related to on chain operations.